ts/src/index.ts

```diff
@@ -84,6 +84,13 @@ function initConfigLocal(config: Config): ProfilerConfig {
   const mergedConfig =
       extend(true, {}, defaultConfig, envSetConfig, envConfig, config);
 
+  // If sourceMapSearchPath is empty array, extend will override its value
```

Comment:
This comment could be clarified a bit - it's not clear what's wrong with the default value from mergedConfig.

Reply:
I tried to clarify the comment more. 

Basically, extend seems to treat a field set with an empty array in the same way it would treat an unset or undefined field -- so this is needed.